pkgs/development/interpreters/erlang/R19.nix

```diff
@@ -14,6 +14,6 @@ mkDerivation {
   ];
 
   prePatch = ''
-    substituteInPlace configure.in --replace '`sw_vers -productVersion`' "''${MACOSX_DEPLOYMENT_TARGET:-10.12}"
+    substituteInPlace configure.in --replace '`sw_vers -productVersion`' "${stdenv.macosVersionMin or ""}"
```

Comment:
Isn't MACOSX_DEPLOYMENT_TARGET set in the setup script? I guess moving it here doesn't hurt, but I'm not sure if it makes it any more reproducible.